components/tableau/index.tsx

```diff
@@ -10,7 +10,7 @@ const TableauNoSSR = dynamic(() => import('@stoddabr/react-tableau-embed-live').
 export const Tableau = (
   { hostUrl, height, width = '100%', toolbar = 'hidden', device = 'default' }: { hostUrl: string, height?: number | string, width?: number | string, toolbar?: 'top' | 'bottom' | 'hidden', device?: 'default' | 'desktop' | 'tablet' | 'phone' }) => {
   return (
-    <div className={`mx-auto w-full bg-slate-100 rounded p-1 my-3 overflow-scroll`}>
+    <div className={`mx-auto w-full bg-slate-100 rounded p-1 my-3 overflow-auto`}>
```

Comment:
I think overflow-scroll looked ok on mac as it hides the scroll bars when not needed, but on other systems it forces them to show (https://tailwindcss.com/docs/overflow#scrolling-in-all-directions). So have changed to auto - has got rid of them on my phone/pc at least.